astroquery/heasarc/core.py

```diff
@@ -121,12 +121,12 @@ def query_object_async(self, object_name, mission, *,
                            cache=True, get_query_payload=False,
                            **kwargs):
         """
-        Query around a specific object within a given mission catalog
+        Query around a specific objects within a given mission catalog
 
         Parameters
         ----------
-        object_name : str
-            Object to query around. To set search radius use the 'radius'
+        object_name : str or iterable
+            Objects to query around. To set search radius use the 'radius'
```

Comment:
Could you specify what the iterable is here?  i.e., the iterable must be an iterable (list, tuple, ...) of strings

Reply:
Can do. I tested with lists and tuples, but should work with any iterable

Reply:
Changed the wording here to ` object_name : str or iterable (list or tuple) of strings`